Source/WebCore/css/parser/CSSParserTokenRange.h

```diff
@@ -85,13 +85,13 @@ class CSSParserTokenRange {
 
     void consumeComponentValue();
 
-    void consumeWhitespace()
+    void consumeWhitespaceLikeTokens()
     {
-        while (peek().type() == WhitespaceToken)
+        while (peek().type() == WhitespaceToken || peek().type() == NewlineToken)
```

Comment:
Might be helpful to add an `isWhitespace` function that takes a type instead of listing the two different types in each place we have to add this.